src/Android/Xamarin.Android/Samples/Geometry/FormatCoordinates/FormatCoordinates.cs

```diff
@@ -0,0 +1,201 @@
+
+        private void Initialize()
+        {
+            // Create the map
+            _myMapView.Map = new Map(Basemap.CreateStreets());
+
+            // Add the graphics overlay to the map
+            _myMapView.GraphicsOverlays.Add(new GraphicsOverlay());
+
+            // Create the starting point
+            MapPoint startingPoint = new MapPoint(0, 0, SpatialReferences.WebMercator);
+
+            // Update the UI with the initial point
+            UpdateUiFromMapPoint(startingPoint);
+
+            // Subscribe to map tap events to enable tapping on map to update coordinates
+            _myMapView.GeoViewTapped += (sender, args) => { UpdateUiFromMapPoint(args.Location); };
+        }
+
+        private void InputTextChanged(object sender, EventArgs e)
+        {
+            // Keep track of the last edited field
+            _lastEdited = (EditText)sender;
+        }
+
+        private void ProcessTextChange(object sender, EventArgs e)
+        {
+            // Hold the entered point
+            MapPoint enteredPoint = null;
+
+            // Update the point based on which textbox sent the event
+            try
+            {
+                switch (_lastEdited.Hint)
+                {
+                    case "Decimal Degrees":
+                    case "Degrees, Minutes, Seconds":
+                        enteredPoint =
+                            CoordinateFormatter.FromLatitudeLongitude(_lastEdited.Text, _myMapView.SpatialReference);
+                        break;
+
+                    case "UTM":
+                        enteredPoint =
+                            CoordinateFormatter.FromUtm(_lastEdited.Text, _myMapView.SpatialReference, UtmConversionMode.LatitudeBandIndicators);
+                        break;
+
+                    case "USNG":
+                        enteredPoint =
+                            CoordinateFormatter.FromUsng(_lastEdited.Text, _myMapView.SpatialReference);
+                        break;
+                }
+            }
+            catch (Exception)
+            {
+                // The coordinate is malformed, return
+                // Sample doesn't handle this because coordinates can be invalid while the user is experimenting
+                return;
+            }
+
+            // Return if getting the point from text failed
+            if (enteredPoint == null)
+            {
+                return;
+            }
+
+            // Update the UI from the MapPoint
+            UpdateUiFromMapPoint(enteredPoint);
+        }
+
+        private void UpdateUiFromMapPoint(MapPoint startingPoint)
+        {
+            // Clear event subscriptions - prevents an infinite loop
+            _UtmEditText.TextChanged -= InputTextChanged;
+            _DmsEditText.TextChanged -= InputTextChanged;
+            _DecimalDegreesEditText.TextChanged -= InputTextChanged;
+            _UsngEditText.TextChanged -= InputTextChanged;
+
+            // Update the decimal degrees textbox
```

Comment:
Maybe change all comments that refer to "textbox" to use "text" instead (works for all platform controls).